nservicebus/messaging/batched-dispatch.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Batched message dispatch
-summary: Describes how NServiceBus collects outgoing operations when processing message in order to dispatch them more efficiently.
+summary: Describes how NServiceBus can collect outgoing operations in order to dispatch them more efficiently.
 component: Core
-reviewed: 2022-08-23
+reviewed: 2022-09-20
 ---
 
-In Versions 6 and above, all outgoing operations that happen as part of processing a message (e.g. commands, responses, events) will be bundled together and passed to the transport after message handling pipeline has completed. This has two benefits:
+Starting from Versions 6, all outgoing operations executed during message processing (e.g. commands, responses, events) are bundled together and handed to the transport only after the message handling pipeline has completed (the [transactional session](/nservicebus/transactional-session) offers similar behavior for outgoing message operations when the session is committed). This has two benefits:
```

Comment:
maybe it needs to be more clear that the transactional session providing similar capabilities outside the pipeline?

```suggestion
Starting from Versions 6, all outgoing operations executed during message processing (e.g. commands, responses, events) are bundled together and handed to the transport only after the message handling pipeline has completed (the [transactional session](/nservicebus/transactional-session) offers similar behavior for `IMessageSession` operations). This has two benefits:
```

Reply:
Discussed with @tmasternak and @SzymonPobiega we are leaving this one out since mentioning `IMessageSession` with the transactional session could make things confusing